pkgs/development/python-modules/gdb-pt-dump/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "gdb-pt-dump";
+  version = "0-unstable-2024-04-01";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "martinradev";
+    repo = "gdb-pt-dump";
+    rev = "50227bda0b6332e94027f811a15879588de6d5cb";
+    hash = "sha256-yiP3KY1oDwhy9DmNQEht/ryys9vpgkFS+EJcSA6R+cI=";
+  };
+
+  build-system = [ poetry-core ];
+
+  pythonImportsCheck = [ "pt" ];
```

Comment:
in next release we will use here: `pt_gdb` and `pt`

Reply:
I think we can only import `pt_gdb` when running inside GDB, which means we can't use `pythonImportsCheck` unfortunately.